crates/primitives/starknet/src/execution/call_entrypoint_wrapper.rs

```diff
@@ -63,14 +64,13 @@ impl CallEntryPointWrapper {
     }
 
     /// Convert to Starknet CallEntryPoint
-    pub fn to_starknet_call_entry_point(&self) -> CallEntryPoint {
+    fn to_starknet_call_entry_point(&self) -> Result<CallEntryPoint, StarknetApiError> {
```

Comment:
impl tryfrom trait

Reply:
there is already try_into this is just a helper internal function